sysid-application/src/main/native/include/sysid/analysis/FeedbackControllerPreset.h

```diff
@@ -103,10 +103,10 @@ constexpr FeedbackControllerPreset kWPILibOld{1.0 / 12.0, 1.0, 50_ms, false,
  *
  * Total delay = 50 ms + 31.5 ms = 81.5 ms.
  */
-constexpr FeedbackControllerPreset kCTRENew{1.0 / 12.0, 0.1, 1_ms, true,
-                                            81.5_ms};
-constexpr FeedbackControllerPreset kCTREOld{1023.0 / 12.0, 0.1, 1_ms, false,
-                                            81.5_ms};
+constexpr FeedbackControllerPreset kCTRECANCoder{1.0 / 12.0, 60.0, 1_ms, true,
+                                                 81.5_ms};
+constexpr FeedbackControllerPreset kCTREDefault{1024.0 / 12.0, 0.1, 1_ms, false,
+                                                81.5_ms};
```

Comment:
Could someone verify that these are the proper settings for the two different types?